gmso/formats/mcf.py

```diff
@@ -0,0 +1,700 @@
+from __future__ import division
+
```

Comment:
Unless you have a good reason to include these 2/3 compatibilities, my preference is not include them at all